src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -511,6 +511,23 @@ namespace Sapphire::Network::Packets::Server
     /* 0018 */ uint64_t targetId;
   };
 
+  /**
+  * Structural representation of the packet sent by the server
+  * to update certain player details / status
+  */
+  struct FFXIVIpcActorControl20F : FFXIVIpcBasePacket< ActorControl20F >
```

Comment:
I don't think ActorControl is a good name for this.
ActorControls are very generic packets, this is way more specific.

Reply:
the opcode is missing in ipcs.h too

Reply:
Oh I thought I had to name it like this because xivmon was naming it `ActorCtrl<Unk:020F>`, something like that. I'll rename it properly and add the opcode to Ipcs.h then 👍

Reply:
that's an actual actorcontrol then which are defined in this file https://github.com/SapphireServer/Sapphire/blob/develop/src/common/Network/CommonActorControl.h